src/intellisense/wrapper.rs

```diff
@@ -0,0 +1,582 @@
+use crate::intellisense::ffi::*;
+use crate::wrapper::Dxc;
+use com_rs::ComPtr;
+use winapi::shared::winerror::HRESULT;
+use winapi::shared::wtypes::BSTR;
+use winapi::shared::ntdef::LPSTR;
+use std::ffi::CString;
+use std::os::raw::c_char;
+
+#[derive(Debug)]
+pub struct DxcIntellisense {
+    inner: ComPtr<IDxcIntelliSense>,
+}
+
+impl DxcIntellisense {
+    pub fn new(inner: ComPtr<IDxcIntelliSense>) -> Self {
```

Comment:
This shouldn't be pub, I don't like exposing the ComPtr<> stuff externally.

Reply:
Fixed